tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -18,8 +18,19 @@ Suite Setup         RHOSi Setup
 Resource            ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
 Resource            ../../../Resources/Common.robot
 
-
 *** Test Cases ***
+Verify Dashbord has no message with NO Component Found
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1493
+    [Documentation]   Verify the service appears in Applications > Enabled
```

Comment:
could you be more specific in the doc? maybe mentioning the bad subscription issue